test/tests/problems/eigen_problem/initial_condition/tests

```diff
@@ -76,4 +76,15 @@
     requirement = "The system shall support eigenvalue calculations with an initial condition including the initial solution vector and the initial eigenvalue."
     slepc = true
   [../]
+
+  [./restart]
```

Comment:
same. You can do the same for the rest of the file or leave it, let's just make the diff clean at least